canal/config.go

```diff
@@ -124,5 +128,7 @@ func NewDefaultConfig() *Config {
 	c.Dump.DiscardErr = true
 	c.Dump.SkipMasterData = false
 
+	c.Logger = log.NewDefault(&log.NullHandler{})
```

Comment:
Can you use the handler from `NewStreamHandler(os.Stdout)` (or also file a PR to siddontang/golog to make newStdHandler public)? so we won't break compatibility and users like you can still change the logger.